MapboxGeocoder/MBPlacemark.swift

```diff
@@ -352,6 +364,15 @@ open class Placemark: NSObject, Codable {
         }
         return String(describing: houseNumber)
     }
+    
+    /**
+     An array of locations that serve as hints for navigating to the placemark.
+     
+     If the `GeocodeOptions.includesRoutableLocations` property is set to `true`, this property contains locations that are suitable to use as a waypoint in a routing engine such as MapboxDirections.swift. Otherwise, if the `GeocodeOptions.includesRoutableLocations` property is set to `false`, this property is set to `nil`.
+     
+     For the placemark’s geographic center, use the `location` property. The routable locations may differ from the geographic center. For example, if a house’s driveway leads to a street other than the nearest street (by straight-line distance), then this property may contain the location where the driveway meets the street. A route to the placemark’s geographic center may be impassable, but a route to the routable location would end on the correct street with access to the house.
+     */
+    @objc open var routableLocations: [CLLocation]?
```

Comment:
I think we should move this property to GeocodedPlacemark, since it doesn’t make sense as a property of QualifyingPlacemark. We’ll need to define an initializer explicitly for GeocodedPlacemark.

/ref https://github.com/mapbox/MapboxGeocoder.swift/pull/145#discussion_r186165700